### ISSUE TITLE
Make 2 parameter continuation for all conductances (gleak, gKs, ...)

### ISSUE THREAD

**Issue by jrieke** [jrieke/drosophila-dynamics#11]

**Comment by jrieke** [comment]
Initial drafts (axis limits and colors not final) - black are saddle-node points, red are Hopf bifurcations:

![2par_gleak](https://cloud.githubusercontent.com/assets/5103165/11321343/750b1294-90bb-11e5-984d-f4a22a76097b.png)
![2par_gks](https://cloud.githubusercontent.com/assets/5103165/11321344/7eee404c-90bb-11e5-857b-926d9a997ac0.png)
![2par_gkf](https://cloud.githubusercontent.com/assets/5103165/11321345/8338b470-90bb-11e5-847a-9d4bdfe1a3de.png)
![2par_gna](https://cloud.githubusercontent.com/assets/5103165/11321347/8a453bc6-90bb-11e5-8ad5-e3c64df33c4d.png)
![2par_gnap](https://cloud.githubusercontent.com/assets/5103165/11321348/8cd650e6-90bb-11e5-940c-f5fd11c7410f.png)

@borismarin I refactored the AUTO code and deleted your original script (it was not compatible any more with the changes I made). Everything can be run through the [Jupyter notebook](https://github.com/jrieke/drosophila-dynamics/blob/master/auto/run.ipynb) now. If you still need your script, the original code is on the auto branch.

**Comment by borismarin** [comment]
TBD
- why only par_x vs I (what about other par combinations not involvig I)?
- homoclinics?
- snics?
- branch of saddle-node for cycles emerging from the Gen. Hopf point

**Comment by jrieke** [comment]
@borismarin I played a bit with other parameter combinations a few days back, but decided not to include them in the thesis. It's already long enough and the conductances are the most interesting thing in my eyes as they represent the strengths of the currents. This way, I can put in some discussion about which role each ion current plays (in terms of excitability, integrator/resonator properties, oscillations, delays, ...), and also relate this to experimental findings where ion channels were blocked/genetically modified. 

Also, I made a few representative (1-parameter) bifurcation diagrams and voltage traces for some of the situations in the plots above.
